### PR TITLE
clicking similar artists now updates events and socials as well, bug fixes

### PR DESCRIPTION
**Pull request by nartexyu**
What was changed:
1. Put your ajax calls into a function so I could call the function again for related artists to prevent having to copy/paste the calls again.
2. Fixed bug where FB/Twitter Logo and Alias' would not up clear if the next related artist didn't have any (you were appending an <a> as a parent and so the .empty()'d applied to the child ID wouldn't apply to it. I added ID's to the appended parents and added code to clear them as well.

### CODE REVIEW

**Comment by nartexyu** [comment]
Also forgot to mention on commit: 791cd48:
1. changed background-img to background so I could darken the artist img by 20% to make the name a little easier to read when the img is brighter
2.  1 comment change

File: homeScript.js [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
+        };
+    };   
+});
 
 
 $("#button-addon2").on("click", function() {
-   var artist = encodeURIComponent($('.form-control').val().toLowerCase());
-    localStorage.setItem('userInput', artist)
+    var artist = encodeURIComponent($('.form-control').val().toLowerCase());
+    localStorage.setItem('lastSsearch', artist)
```
Comment (ma-wong) [inline homeScript.js:15]: typo? did you mean `lastSearch`?

File: homeScript.js [hunk 2]
```diff
@@ -1,21 +1,19 @@
...
     if (artist!=="") {
-        window.location.href="./index.html"
-    }
+        window.location.href="index.html"
```
Comment (ma-wong) [inline homeScript.js:17]: Does this make a difference?

File: index.html [hunk 3]
```diff
@@ -37,7 +37,7 @@
             <!-- Artist Image and Name - desktop & mobile -->
             <div class="col-lg-9 col-sm-12 jumbo-responsive">
                 <div class="row">
-                    <div class="jumbotron jumbotron-fluid jumbotron-image col-12">
+                    <div class="jumbotron jumbotron-fluid jumbotron-image col-12" id="jumbotron">
```
Comment (ma-wong) [inline index.html:40]: Does this do anything? [here](
https://github.com/mfitzwinick/binary-babes/pull/30/files/a080c9e213eb5123e2cb818d99eeec0bdc89448c#diff-ed3ee7e0beea2498ff3b8ca85973d122fc6fa3d585d62b5807ec034d0cf076b3R128) the "id=jumbotron" that you created isn't called anywhere in the js file. youre still using the class i made called ".jumbotron-image" to style it which does the same thing

File: script.js [hunk 4]
```diff
@@ -113,12 +124,14 @@ $(document).on('keypress',function(e) {
...
             var jumboImg = res1.image_url;
-            console.log(jumboImg);
-            $('.jumbotron-image').attr('style', 'background-image:' + "url(" + jumboImg + ")");
+            // console.log(jumboImg);
+            $('.jumbotron-image').attr('style', 'background: '+ "linear-gradient(rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2)), " + "url(" + jumboImg + ")");
 
              // if artist has a fb page, a clickable fb icon will appear in the social tab and redirects you in a new tab
             if (res1.facebook_page_url === '') {
                 console.log('No Facebook link available');
+                $("fb-logo").empty();
```
Comment (ma-wong) [inline script.js:133]: i already added these in my updated code from last night

File: script.js [hunk 5]
```diff
@@ -129,12 +142,83 @@ $(document).on('keypress',function(e) {
...
+        var aliasArray = dater.message.body.artist_list[0].artist.artist_alias_list;
+        // console.log(dater.message.body.artist_list[0].artist.artist_alias_list)
+        if (aliasArray === []) {
+            console.log('Artist has no known alias');
+            $('#alias-list').empty();
+            return;
+        }
+        else {
+            $('#alias-list').text('This artist may also go by: ');
+            $('#alias-list').attr('style', 'margin-bottom:8px');
```
Comment (ma-wong) [inline script.js:211]: i already mimicked the style of the discography page for the rest of the events in my most recent changes

File: script.js [hunk 6]
```diff
@@ -129,12 +142,83 @@ $(document).on('keypress',function(e) {
...
+            });
+            $('#twitter-logo').wrap($('<a id = "twitterA"/>').attr({href:twitterURL, target:'_blank'})).parent();
+        };
+
+        // Artist Alias
+        var aliasArray = dater.message.body.artist_list[0].artist.artist_alias_list;
+        // console.log(dater.message.body.artist_list[0].artist.artist_alias_list)
+        if (aliasArray === []) {
+            console.log('Artist has no known alias');
+            $('#alias-list').empty();
```
Comment (ma-wong) [inline script.js:206]: this will only remove the header but not the alias of the previous artist

File: homeScript.js (hunk 1 above)
Comment (nartexyu) [reply homeScript.js:15]: oops that was an accident u are correct

File: homeScript.js (hunk 2 above)
Comment (nartexyu) [reply homeScript.js:17]: uh i have no clue tbh i think both worked

File: index.html (hunk 3 above)
Comment (nartexyu) [reply index.html:40]: oh i was going to do something with it but ended up not lol forgot to remove

**Pull request closed without merging**